packages/flutter_tools/lib/src/web/web_device.dart

```diff
@@ -204,19 +204,45 @@ class WebDevices extends PollingDeviceDiscovery {
 
 }
 
+const String _klinuxExecutable = 'google-chrome';
+const String _kMacOSExecutable = '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome';
+const String _kWindowsExecutable = r'Google\Chrome\Application\chrome.exe';
+final List<String> _kWindowsPrefixes = <String>[
+  platform.environment['LOCALAPPDATA'],
+  platform.environment['PROGRAMFILES'],
+  platform.environment['PROGRAMFILES(X86)'],
+];
+
 // Responsible for launching chrome with devtools configured.
 class ChromeLauncher {
   const ChromeLauncher();
 
-  static const String _kMacosLocation = '/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome';
-
   Future<void> launch(String host) async {
     if (platform.isMacOS) {
       return processManager.start(<String>[
-        _kMacosLocation,
+        _kMacOSExecutable,
```

Comment:
Consider failing more gracefully than with the exception that this will throw if _kMacOSExecutable doesn't exist.